robotino_description/urdf/base.urdf.xacro

```diff
@@ -5,22 +5,86 @@
 
   <xacro:macro name="base" params="name">
   
-    <!--link name="${name}_footprint">
-   <visual>
+   <joint name="origin_joint" type="prismatic">
```

Comment:
@shangl Thank you so much for the explanation! Based on your pull request I tried to create a base model that could be spawned on gazebo and reused for all the robots, With this version all the robots, with or without arm will have a base_jointx/y/z if you dont have an arm the joints are fixed if you have an arm you can use them to plan and move.
https://github.com/squirrel-project/squirrel_common/pull/142/files  the gazebo and transmission file are only needed for gazebo, I added some comments to understand the changes on the urdf file.

I also adapted robotino_bringup  https://github.com/squirrel-project/squirrel_robotino/pull/152 , for alll the robots the base_linkz is the reference frame of the base and all of them spawn the static transformation between odom and origin.

 The tf trees  

8dof:
[frames.pdf](https://github.com/squirrel-project/squirrel_common/files/495349/frames.pdf)

without arm:
[frames_non8dof.pdf](https://github.com/squirrel-project/squirrel_common/files/495350/frames_non8dof.pdf)